rurusetto/wiki/templates/wiki/status.html

```diff
@@ -65,7 +77,13 @@ <h1><img src="{{ ruleset.0.icon.url }}" alt="{{ ruleset.0.name }}" width="40" he
     </div>
     {% elif ruleset.1 == 'patreon' %}
     <div class="row" data-aos="fade-up" data-aos-duration="600" data-aos-once="true">
-        <h1><img src="{{ ruleset.0.icon.url }}" alt="{{ ruleset.0.name }}" width="40" height="40" class="rounded-circle hvr-icon"> {{ ruleset.0.name }}</h1>
+        <h1>
+            {% if user.config.theme == 'light' %}
+            <img src="{{ ruleset.0.icon.url }}" alt="{{ ruleset.0.name }}" width="40" height="40" class="rounded-circle hvr-icon"> 
+            {% else %}
+            <img src="{{ ruleset.0.dark_icon.url }}" alt="{{ ruleset.0.name }}" width="40" height="40" class="rounded-circle hvr-icon"> 
+            {% endif %}            
+            {{ ruleset.0.name }}</h1>
```

Comment:
```suggestion
        <h1>
            {% if user.config.theme == 'light' %}
            <img src="{{ ruleset.0.light_icon.url }}" alt="{{ ruleset.0.name }}" width="40" height="40" class="rounded-circle hvr-icon"> 
            {% else %}
            <img src="{{ ruleset.0.icon.url }}" alt="{{ ruleset.0.name }}" width="40" height="40" class="rounded-circle hvr-icon"> 
            {% endif %}            
            {{ ruleset.0.name }}
        </h1>
```
Change like above and move `</h1>` tag to newline is better.